### PR TITLE
MWPW-140438 - Add martech metadata tool to Preflight

### PR DESCRIPTION
**Pull request by elan-tbx**
This PR adds a new "Martech" tab to the Preflight window that automatically generates the `martech-metadata` block by scanning the currently previewed page. Authors can click the "copy" button, under this new tab, and paste the block into the related Word document. This solves the need to have the `martech-metadata` block created manually.

The `martech-metadata` block was previously merged in #1609, but manually building the table can be a cumbersome process. 

**Testing notes**
* Make sure when trying to test this tool that you have the URL param `milolibs=ebartholomew-martech-tool` in your address bar.

Resolves: [MWPW-140438](https://jira.corp.adobe.com/browse/MWPW-140438)

**Test URLs:**

Milo:
- Before: https://main--milo--adobecom.hlx.page/products/indesign?martech=off
- After: https://ebartholomew-martech-tool--milo--adobecom.hlx.page/products/indesign?martech=off

CC:
- Before: https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?martech=off
- After: https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?martech=off&milolibs=ebartholomew-martech-tool

DC:
- Before: https://main--dc--adobecom.hlx.page/acrobat/features/compare-pdfs?martech=off
- After: https://main--dc--adobecom.hlx.page/acrobat/features/compare-pdfs?martech=off&milolibs=ebartholomew-martech-tool

Blog:
- Before: https://main--blog--adobecom.hlx.page/en/publish/2024/01/16/adobe-introduces-major-new-audio-updates-premiere-pro-ahead-2024-sundance-film-festival?martech=off
- After: https://main--blog--adobecom.hlx.page/en/publish/2024/01/16/adobe-introduces-major-new-audio-updates-premiere-pro-ahead-2024-sundance-film-festival?martech=off&milolibs=ebartholomew-martech-tool

### CODE REVIEW

**Review by vgoodric — APPROVED** [review]

**Comment by rgclayton** [comment]
FYI. For testing all you need to do is update your sidekick to point to the feature branch this PR is using.
replace `main` with FB name.
image

**Comment by rgclayton** [comment]
Ni AX: There's no feedback when clicking the "copy table" button. It made me think it didn't copy. 
Maybe a message below the button that says "copied table. Paste table in your Word doc to finalize" - something along those lines. Maybe the button slightly changes color on focus too? Just trying to minimize the feeling that the copy didn't work.

File: libs/blocks/preflight/panels/martech.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+function getTable(strings) {
+  const table = document.createElement('table');
+  table.setAttribute('border', 1);
+  const headerRow = document.createElement('tr');
+  headerRow.append(createTag('th', { colspan: 2, style: 'width: 100%' }, 'martech metadata'));
+  table.append(headerRow);
+  strings.forEach((str) => {
+    const tr = document.createElement('tr');
+    tr.append(createTag('td', { colspan: 1 }, createTag('h3', {}, str)));
+    tr.append(createTag('td', { colspan: 1 }, createTag('h3', { 'data-ccp-parastyle': 'DNT' }, str)));
```
Comment (rgclayton) [inline libs/blocks/preflight/panels/martech.js:15]: Nit: Any reason you're using h3's?

If no real reason I could see bumping the font size down (16px?) so the table isn't as long.

Comment (elan-tbx) [reply libs/blocks/preflight/panels/martech.js]: Under the hood in Word, the DNT style produces an H3. So this is just to try and pre-align with it before dropping the block in there.

**Review by rgclayton — APPROVED** [review]

**Pull request merged**